db/models/taskResult.go

```diff
@@ -70,9 +70,11 @@ type TsHopResult struct {
 }
 
 type TsTracerouteResult struct {
-	DestinationAdress [4]byte
-	Hops              []TsHopResult
+	TsSensorTaskBase
+	DestinationAdress net.IP        `gorm:"type:inet"`
+	Hops              []TsHopResult `gorm:"type:jsonb"` // question: is this efficient?
```

Comment:
@arthureld Do we really need need this TsTracerouteResult.Hops column? Basically we have 2 tables for storing the results: TsTracerouteResult and TsHopResult. We will be able to select/join the result from both based on the TaskID which is part of the TsSensorTaskBase(logically not based on real fkeys). Also please try to run the migrations and to store some real task results into the db(if you did not already:)